src/web/page/mod.rs

```diff
@@ -3,14 +3,29 @@ pub(crate) mod web_page;
 
 pub(crate) use templates::TemplateData;
 
-use serde::Serialize;
+use crate::f_a::IconStr;
+use serde::ser::{Serialize, SerializeStruct, Serializer};
 
-#[derive(Debug, Copy, Clone, PartialEq, Eq, Serialize)]
+#[derive(Debug, Copy, Clone, PartialEq, Eq)]
 pub(crate) struct GlobalAlert {
     pub(crate) url: &'static str,
     pub(crate) text: &'static str,
     pub(crate) css_class: &'static str,
-    pub(crate) fa_icon: &'static str,
+    pub(crate) fa_icon: crate::f_a::icons::IconTriangleExclamation,
```

Comment:
shouldn't this be `Box<dyn font_awesome_as_a_crate::IconStr>`? 

And I'm probably missing something around `Serialize` and how rinja accesses the values. 

Do you actually need `Serialize` on `fa_icon` when you use it as such? 

```
{{- global_alert.fa_icon.clone()|fas(false, false, "") }}
```

Reply:
I don't need `Serialize` for templates. If it's not needed at all, I'll simply remove it.

Reply:
@GuillaumeGomez thinking about this again, 

let's keep it _very_ simple. 
I never used that feature, though we might at some point. 

You can just use `IconTriangleExclamation` directly in the template. 

About `Serialize`, if it works, you can remove it

Reply:
Removing code makes me very happy. :)